static/admin.css

```
@@ -1342,3 +1375,758 @@ body[data-page="admin"] [data-quota-runway-row] [data-quota-runway-details] p {
+body[data-page="admin"] {
+  --surface: rgba(17, 20, 24, 0.94);
+  --surface-2: rgba(255, 255, 255, 0.07);
+  --surface-3: rgba(255, 255, 255, 0.11);
+  --text: #f5f7fa;
+  --muted: #b5bdc8;
+  --muted-2: #8e98a5;
+  --accent: #f5f7fa;
+  --accent-ink: #080a0d;
+  --danger: #ffaaa8;
+  --shadow-soft: 0 16px 36px rgba(0, 0, 0, 0.24);
+  background-color: #080a0d;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)